### PR TITLE
Update note

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/10032

### CODE REVIEW

File: Teams/direct-routing-plan.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Direct Routing lets you connect a supported, customer-provided Session Border Co
   > Skype for Business Online also lets you pair a customer-provided SBC, but this requires an on-premises Skype for Business Server deployment or a special edition of Skype for Business, called Cloud Connector, in between the SBC and the Microsoft Cloud. This scenario is known as hybrid voice. In contrast, Direct Routing allows a direct connection between the supported SBC and the Microsoft Cloud.
 
 > [!Important]
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md:38]:
```suggestion
> [!IMPORTANT]
```

File: Teams/direct-routing-plan.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Direct Routing lets you connect a supported, customer-provided Session Border Co
...
-> Cloud Connector Edition will retire July 31, 2021 along with Skype for Business Online. Once your organization has upgraded to Teams, learn how to connect your on-premises telephony network to Teams using [Direct Routing](direct-routing-landing-page.md). 
+> Cloud Connector Edition was retired on July 31, 2021 along with Skype for Business Online. Once your organization has upgraded to Teams, learn how to connect your on-premises telephony network to Teams using this page.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md:39]:
```suggestion
> Cloud Connector Edition was retired on July 31, 2021, along with Skype for Business Online. Once your organization has upgraded to Teams, learn how to connect your on-premises telephony network to Teams using this article. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**